indico/modules/rb/client/js/components/SidebarMenu.jsx

```diff
@@ -165,6 +180,7 @@ export default connect(
     hasOwnedRooms: userSelectors.hasOwnedRooms(state),
   }),
   dispatch => ({
+    dispatchAction: dispatch,
```

Comment:
If you have a better idea on how to do this (allow plugins to dispatch actions), you're welcome to share it!

Reply:
`const dispatch = useDispatch()` inside the plugin's component? not sure if importing stuff from react-redux in plugins works though... and i guess this type of redirection of calling hooks would be pretty nasty.

so probably your option is already the way to go there (i'd just keep the `dispatch` name and add a comment mentioning that we want to provide it to plugins so nobody removes it as being unused)

Reply:
The problem with the `dispatch` name is that along the way it got "stolen" by one of the HOCs. But I think we don't need to actually steal it (probably a remnant from some other thing).

Reply:
Added the comment too.